src/iscsi.c

```diff
@@ -2214,7 +2268,12 @@ isboot_action(struct cam_sim *sim, union ccb *ccb)
 				break;
 			}
 		}
-		isboot_scsi_io(sim, ccb);
+		taskp = isboot_malloc(sizeof(*taskp));
```

Comment:
Allocations in I/O path are bad, but allocations panic'ing on error are even worse.  struct ccb_hdr has sim_priv field of two pointers, just enough to store TAILQ_ENTRY in it.